tests/ui/arch/any.rs

```diff
@@ -1,9 +1,26 @@
 // build-pass
 
+#![feature(repr_simd)]
+
 use spirv_std::spirv;
+use spirv_std::{scalar::Scalar, vector::Vector};
+/// HACK(shesp). Rust doesn't allow us to declare regular (tuple-)structs containing `bool` members
```

Comment:
```suggestion
use spirv_std::{scalar::Scalar, vector::Vector};

/// HACK(shesp). Rust doesn't allow us to declare regular (tuple-)structs containing `bool` members
```
(for consistency with the other test - feel free to just copy-paste between them)